docs/getting-started/upgrade/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Upgrade Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/upgrade/"
+---
+
+You can manage upgrades to Shopify App CLI with the package manager for your platform.
+
+### Ruby gem
+
+```console
+$ gem update shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+```console
+$ apt update && apt upgrade shopify-cli
```

Comment:
Suggest to break out these two commands, and put `sudo` in front of each of them.